site/src/theme/darkBlue/roles.ts

```diff
@@ -72,6 +62,16 @@ export default {
 			text: colors.white,
 		},
 	},
+	info: {
+		background: colors.gray[950],
+		outline: colors.gray[400],
+		text: colors.gray[50],
+		fill: {
+			solid: colors.gray[500],
+			outline: colors.gray[600],
+			text: colors.white,
+		},
+	},
```

Comment:
I know that Tailwind technically saturates their colors so the problem won't be as pronounced, but is gray on blue going to look washed out?

Reply:
I think they look fine, and we can always tweak it going forward.